plenum/test/replica_removing/test_replica_removing.py

```diff
@@ -148,7 +150,7 @@ def test_unordered_request_freed_on_replica_removal(looper,
         assert len(node.requests) == 1
         forwardedToAfter = next(iter(node.requests.values())).forwardedTo
         assert forwardedToAfter == forwardedToBefore - 1
-        chkChkpoints(txnPoolNodeSet, 0)
+        chkChkpoints(txnPoolNodeSet, 1)
```

Comment:
Why is it 1 now?

Reply:
Because at the start of the test, we already have a new checkpoint after ordering transactions for the Audit Ledger.